source/pug/partials/hero-unit.pug

```diff
@@ -1,5 +1,5 @@
-header
-  .header-content.container
+header.page-header
```

Comment:
a page can have more than one `<header>`. If we wanna style this particular header only, assign it a class name and apply CSS just on that class.